### ISSUE TITLE
URL to candybox is dead

### ISSUE THREAD

**Issue by mudream4869** [doublespeakgames/adarkroom#693]
https://github.com/doublespeakgames/adarkroom/blob/main/index.html#L10

```html
        <!--  
		A Dark Room (v1.4)
		==================
		
		A minimalist text adventure by Michael Townsend and all his friends.
		Inspired by Candy Box (http://candies.aniwey.net/)
		Contribute on GitHub! (https://github.com/doublespeakgames/adarkroom/)
	-->
```

Maybe this? https://github.com/candybox2/candybox

**Comment by esaruoho** [comment]
hi @avengah @mudream4869  - i've created a PR for this fix.
https://github.com/doublespeakgames/adarkroom/pull/722

**Comment by AverseABFun** [comment]
This issue should be closed via #722 (btw not part of doublespeakgames anyone) @doublespeakgames

**Comment by esaruoho** [comment]
@mudream4869 hi, i fixed your issue, it's been merged in, has landed, has been fixed, so this ticket can be closed. can you close this ticket, please?
